backtesting/backtesting.py

```diff
@@ -511,10 +524,12 @@ def __init__(self, broker: '_Broker', size: int, entry_price: float, entry_bar):
         self.__exit_bar: Optional[int] = None
         self.__sl_order: Optional[Order] = None
         self.__tp_order: Optional[Order] = None
+        self.__order_id = id
 
     def __repr__(self):
         return f'<Trade size={self.__size} time={self.__entry_bar}-{self.__exit_bar or ""} ' \
-               f'price={self.__entry_price}-{self.__exit_price or ""} pl={self.pl:.0f}>'
+               f'price={self.__entry_price}-{self.__exit_price or ""} pl={self.pl:.0f}' \
+               f'{" id="+str(self.__order_id) if self.__order_id is not None else ""}>'
```

Comment:
```suggestion
               f' id={self.__order_id if self.__order_id is not None else ""}>'
```
... if not moving it out of the return statement / making it optional altogether.

Reply:
But wouldn't that potentially leave a hanging `id=` ?

Reply:
Ah, that's exactly correct. Never mind then. :flushed: